src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -106,6 +106,13 @@ MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
  */
 - (void)controller:(MTRDeviceController *)controller
     readCommissioningInfo:(MTRProductIdentity *)info MTR_AVAILABLE(ios(17.0), macos(14.0), watchos(10.0), tvos(17.0));
+
+/**
+ * Notify the delegate when the suspended state changed of the controller, after this happens
```

Comment:
Two things:

1) "the suspended state of the controller changes".
2) "The claim that after that the controller is in the specified state does not happen to be true; it could be in the other state but we have just not gotten our async notification for that yet....